tests/Aspire.EndToEnd.Tests/Aspire.EndToEnd.Tests.csproj

```diff
@@ -2,10 +2,62 @@
 
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
+
+    <!-- no docker support on helix/windows yet -->
+    <RunTestsOnHelix Condition="'$(OS)' != 'Windows_NT'">true</RunTestsOnHelix>
+    <SkipTests Condition="'$(OS)' == 'Windows_NT'">true</SkipTests>
```

Comment:
Can we just say `RunTestsOnHelix=true` all the time, and then do these checks inside the tests? Similar to how I did it in RabbitMQ in https://github.com/dotnet/aspire/pull/2345. It just feels cleaner that way.

Reply:
Yes we can do it like that also. The reason for "also" is that if we don't have any tests that would get run, then not skipping the whole thing would mean:
1. installing sdk+workload - extra step on CI
2. sending this work item to helix with the big payload (sdk) for nothing to run, which dnceng kinda frowns upon.

I think we can add the attributes also, and for the specific case (windows) where nothing can run, we skip in the project file too. Thoughts?

Reply:
How long until we are able to run these tests on Windows?